Automattic/woocommerce-payments
# Add placeholder TaskList component to the Overview page.

Fixes #1656.

#### Changes proposed in this Pull Request



Add an empty task list to the Overview page, to be populated in other PRs.



This PR seeks to add an empty task list to the Overview page. Implementation of tasks will be in other PRs. To accomplish this goal:

* `@woocommerce/experimental` was added as a dependency (contains the List components)
* A new `client/overview/task-list` component was created
* The import order of `@wordpress/base-style` stylesheets was tweaked to prefer the non-React-native styles (see screenshots below)

| Before @import order change | After @import order change |
|--------|-------|
| <img width="306" alt="Screen Shot 2021-05-11 at 1 52 51 PM" src="https://user-images.githubusercontent.com/63922/117878657-31af2380-b263-11eb-8eca-8733921d6f3f.png"> | <img width="332" alt="Screen Shot 2021-05-11 at 1 57 22 PM" src="https://user-images.githubusercontent.com/63922/117878899-7fc42700-b263-11eb-90d5-8124efb832df.png"> |

❕ I spot checked a few areas that I'm familiar with to make sure no styling broke, but someone with more familiarity with the app should verify.





#### Testing instructions





* Enable the overview page
* Verify the overview page isn't broken
* Optionally verify there is a `<TaskList>` component in the React tree (use devtools)

-------------------
